rudder-core/src/main/scala/com/normation/cfclerk/services/impl/SystemVariableSpecServiceImpl.scala

```diff
@@ -210,6 +210,10 @@ class SystemVariableSpecServiceImpl extends SystemVariableSpecService {
                                           , multivalued = false
                                           , isUniqueVariable = true
       )
+//    , SystemVariableSpec("RUDDER_NODE_GROUPS" , "The array of group_UUID => group_NAME for the node, in a formatted string"
```

Comment:
This needs to be added in anothe PR (but since it's commented it's not that important ...) 

Just dead code until next PR